### PR TITLE
VLC Cheat Sheet

### PR DESCRIPTION
**Pull request by codenirvana**
**What does your Instant Answer do?**
A cheat sheet that lists useful VLC keyboard shortcuts.

**What problem does your Instant Answer solve (Why is it better than organic links)?**
It summarizes all the VLC shortcuts so you don't have to go through several links/sources.

**What is the data source for your Instant Answer? (Provide a link if possible)**
[Videolan](https://wiki.videolan.org/Hotkeys_table) official wiki website.

**Why did you choose this data source?**
They list all the most commonly used VLC keyboard shortcuts.

**What are some example queries that trigger this Instant Answer?**
* VLC keyboard shortcuts
* VLC cheat sheet

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
VLC media player users.

**Which existing Instant Answers will this one supersede/overlap with?**
None

**Are you having any problems? Do you need our help with anything?**
No

**What are the terms of use for the API? Will DuckDuckGo need specific authorization (e.g. an API key)? Are there any costs associated with API usage?**
No

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
VLC-keyboard-shortcuts

-----
IA Page: https://duck.co/ia/view/vlc_cheat_sheet

### CODE REVIEW

**Comment by moollaza** [comment]
@codenirvana so sorry for the delay! This one fell through the cracks.

As always, thanks a lot for contributing. We'll review this and give you any feedback :+1:

File: share/goodie/cheat_sheets/vlc.json [hunk 1]
```diff
@@ -0,0 +1,188 @@
+{
+    "id": "vlc_cheat_sheet",
+    "name": "VLC Keyboard Shortcuts",
+    "description": "VLC Media Player Hotkeys",
+    "metadata": {
+        "sourceName": "Videolan",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/vlc.json]: Nit: Let's correct the casing: `VideoLAN`

File: share/goodie/cheat_sheets/vlc.json [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+            "key":"Space",
+            "val":"Play/Pause"
+          }, {  
+            "key":"V",
+            "val":"Subtitles cycle/off"
+          }, {  
+            "key":"B",
+            "val":"Audio track cycle"
+          }, {  
+            "key":"[Ctrl] [↑] / [↓]",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/vlc.json]: These should be wrapped in parenthesis:

`[Ctrl] ([↑]/[↓])`

File: share/goodie/cheat_sheets/vlc.json [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+            "val":"Volume or Position"
+          }, {  
+            "key":"Right click",
+            "val":"Local menu (play controls, audio/video)"
+          }],
+        "Navigation": [{  
+            "key":"[Ctrl] [T]",
+            "val":"Goto / jump to time"
+          }, {  
+            "key":"[Shift] [←] / [→]",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/vlc.json]: Same here, all these sub-alternatives, should be wrapped in `()`

File: share/goodie/cheat_sheets/vlc.json [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+            "key":"D",
+            "val":"Show movie path"
+          }, {  
+            "key":"F1",
+            "val":"Show Help"
+          }, {  
+            "key":"F11",
+            "val":"Window fullscreen"
+          }, {  
+            "key":"[Alt] [F4] / [Ctrl] [Q]",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/vlc.json]: These should be listed as two entries. The second one should mention `(alternative)` in it's description:

```json
{  
  "key":"[Alt] [F4]",
  "val":"Quit VLC"
},{
  "key":"[Ctrl] [Q]",
  "val":"Quit VLC (alternative)"
}
```

**Comment by moollaza** [comment]
@codenirvana this is looking great! Just a couple fixes to be made, please :smiley:

File: share/goodie/cheat_sheets/vlc.json [hunk 5]
```diff
@@ -0,0 +1,188 @@
+{
+    "id": "vlc_cheat_sheet",
+    "name": "VLC Keyboard Shortcuts",
+    "description": "VLC Media Player Hotkeys",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/vlc.json]: I think we can remove this

File: share/goodie/cheat_sheets/vlc.json [hunk 6]
```diff
@@ -0,0 +1,188 @@
+{
+    "id": "vlc_cheat_sheet",
+    "name": "VLC Keyboard Shortcuts",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/vlc.json]: Would update this to `VLC Media Player Keyboard Shortcuts`

**Comment by codenirvana** [comment]
@moollaza Done!

File: share/goodie/cheat_sheets/vlc.json [hunk 7]
```diff
@@ -0,0 +1,190 @@
...
+            "val":"Current from beginning/Previous in playlist"
+          }],
+        "Browsing & Playback Adjustments": [{  
+            "key":"[Ctrl] [D]",
+            "val":"Open disc menu"
+          }, {  
+            "key":"[Ctrl] [F]",
+            "val":"Open folder (browse folder menu)"
+          }, {  
+            "key":"[Ctrl] [R] / [S]",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/vlc.json]: Should be `[Ctrl] ([R]/[S])`

File: share/goodie/cheat_sheets/vlc.json [hunk 8]
```diff
@@ -0,0 +1,190 @@
...
+            "key":"[Ctrl] [C]",
+            "val":"Open capture device"
+          }, {  
+            "key":"[Ctrl] [L]",
+            "val":"Open playlist"
+          }, {  
+            "key":"[Ctrl] [Y]",
+            "val":"Save playlist"
+          }, {  
+            "key":"[Ctrl] [I] / [J]",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/vlc.json]: Should be `[Ctrl] ([I]/[J])`

File: share/goodie/cheat_sheets/vlc.json [hunk 9]
```diff
@@ -0,0 +1,190 @@
...
+            "key":"=",
+            "val":"Normal"
+          }, {  
+            "key":"A",
+            "val":"Aspect ratio"
+          }, {  
+            "key":"C",
+            "val":"Crop screen"
+          }, {  
+            "key":"[G] / [H]",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/vlc.json]: Should be two separate entries. The latter should mention `(alternative)` in the description

**Comment by codenirvana** [comment]
@moollaza Updated!

**Comment by moollaza** [comment]
@codenirvana LGTM :+1: 

Thanks a lot, and great work :smile:

**Comment by moollaza** [comment]
@codenirvana congrats! It's live: https://duckduckgo.com/?q=vlc+cheat+sheet&ia=cheatsheet

Sorry for the delayed notice, it's actually been live for a little while!

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Pull request merged**